frame/nfts/src/types.rs

```diff
@@ -172,3 +171,89 @@ pub struct PriceWithDirection<Amount> {
 	/// A direction (send or receive).
 	pub(super) direction: PriceDirection,
 }
+
+// Support for up to 64 user-enabled features on a collection.
+#[bitflags]
```

Comment:
I dont trust this macro. Could you add some test that checks that the values are indeed exclusive? So that they have no overlapping bits?

Reply:
this is a good point, we should axe enumflags2 crate in favour of bitflags crate as that's more commonly used in substrate. less deps is better security.

Reply:
We already use that macro in the identity pallet, it's not new here